core/src/main/java/edu/wpi/grip/core/http/GetHandler.java

```diff
@@ -0,0 +1,13 @@
+
+package edu.wpi.grip.core.http;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Handler for creating responses to incoming HTTP {@code GET} requests.
+ */
+public interface GetHandler {
```

Comment:
~~Yes, functional interface.~~

 `@FunctionalInterface` is just a marker annotation, it doesn't affect it being used as a lambda